### PR TITLE
Add support for analytics init options and post setup commands

### PR DESCRIPTION
**Pull request by kiowa**
This will enable cross domain support #6

### CODE REVIEW

File: src/analytics.js [hunk 1]
```diff
@@ -119,7 +120,10 @@ export class Analytics {
 		document.querySelector('body').appendChild(script);
 
 		window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
-		ga('create', id, 'auto');
+		ga('create', id, 'auto', options);
+		for (var cmd of cmds) {
```
Comment (miguelzakharia) [inline src/analytics.js:124]: Would it be better to have this follow the structure of the other GA features? `options` can stay as an arg for `init()`, but `cmds` will be moved to the `options` arg of `attach()`. For example:

`init(id, options) {...}`

```
attach(options = defaultOptions) {
    ...
    this._attachLinker(options.linker.cmds);
    this._attachClickTracker();
    this._attachPageTracker();
}
```

```
_attachLinker(cmds) {
    if(!this._options.linker.enabled) { return; }

    ga('require', 'linker');
    for (var cmd of cmds) {
        ga(cmd[0], cmd[1]);
    }
}
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aurelia-google-analytics",
-  "version": "1.0.3",
+  "version": "1.0.4",
```
Comment (miguelzakharia) [inline package.json]: I think this would bump version to 1.1.0 (Breaking.Feature.Fix)

**Comment by miguelzakharia** [comment]
Thank you for contributing!

**Comment by kiowa** [comment]
Anything missing for you to merge this pull request @miguelzakharia?

**Comment by miguelzakharia** [comment]
Nothing missing, but I made a suggestion on `src/analytics.js` that would being the changes in line with the other features.

**Review by martijnmelchers — APPROVED** [review]

**Pull request closed without merging**